content/en/logs/guide/logs-rbac.md

```diff
@@ -0,0 +1,431 @@
+[5]: /api/v2/logs-restriction-queries/#get-restriction-query-for-a-given-role
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## Restrict access to Log Assets
+
+This section details how to grant `ACME Admin` role members the permission to interact with ACME Log Assets (namely Log Pipelines, Log Indexes, and Log Archives). 
+
+This ensures that:
+
+* `ACME Admin` members (and only `ACME Admin` members) can interact with ACME Log Assets. 
+* Neither `ACME Admin` nor `ACME User` members can interfere with assets from other teams.
+* Neither `ACME Admin` nor `ACME User` members can interfere with higher level "Admin" configurations, such as which logs flow into their assets, budget limitations, or [Log Access Restriction rules](#restrict-access-to-logs).
+
+
+As a good practice for maximum granularity and easier maintainability, you should **not** grant other roles the permission to edit the ACME Log Assets. Instead, consider adding (some) users from those other roles to the `ACME Admin` role as well.
+
+
+### Log Pipelines
+
+Create one [pipeline][60] for `team:acme` logs. Assign the [Write Processor][70] permission to members of `ACME Admin`, but **scope** that permission to this ACME "root" pipeline. 
+
+{{< img src="logs/guide/rbac/pipelines.png" alt="ACME Pipeline"  style="width:60%;">}}
+
+
+### Log Indexes
+
+Create one or multiple [indexes][61] for `team:acme` logs. Multiple indexes can be valuable if ACME team needs fine-grained budget control (for instance, indexes with different retentions, or indexes with different quotas). Assign the [Write Exclusion Filters][71] permission to members of `ACME Admin`, but **scope** that permission to these ACME Index(es). 
+
+{{< img src="logs/guide/rbac/indexes.png" alt="ACME Indexes"  style="width:60%;">}}
+
+
+### Log Archives
+
+#### Read Log Archives
+
+Create one or multiple [archives][62] for `team:acme` logs. Assign the [Read Archives][72] permission to members of `ACME Admin`, but **scoped** to that ACME Archive(s).
+
+{{< img src="logs/guide/rbac/archives.png" alt="ACME Archives"  style="width:60%;">}}
+
+Multiple archives can be useful if you have different lifecycle policies depending on logs (for instance, for production and staging logs). Keep in mind that rehydration is intended to work for only one archive at a time, though you can trigger multiple rehydrations on multiple archives at once. 
+
+#### Write Historical Views
+
+Assign the [Write Historical View][73] permission to members of `ACME Admin`. This permission grants the ability to perform rehydrations. 
+
+**Optionally**, set up your Log Archives so that all logs rehydrated from that archive will eventually have the `team:acme` tag, whether whether or not they had the tag in the archive. This option enables you to enforce consistency with your existing restriction policies, as well as to safely remove deprecated restrictions that correspond to no more logs flowing in Datadog or indexed in Datadog.
```

Comment:
```suggestion
**Optionally**, set up your Log Archives so that all logs rehydrated from that archive will eventually have the `team:acme` tag, whether or not they had the tag in the archive. This option enables you to enforce consistency with your existing restriction policies, as well as to safely remove deprecated restrictions that correspond to no more logs flowing in Datadog or indexed in Datadog.
```